lib/internal/http2/compat.js

```diff
@@ -155,23 +167,17 @@ class Http2ServerRequest extends Readable {
   }
 
   get closed() {
-    const state = this[kState];
-    return Boolean(state.closed);
+    return this[kState].closed;
```

Comment:
I think `kState` can be nulled, shouldn't we protect this?

Reply:
I don't think there's ever a situation where that can happen (I reviewed all of the code in compat) and since it's accessed via a Symbol, users would have to really go out of their way to null it.

Reply:
Ok, good catch!